pkg/deploy/api/v1/conversion_generated.go

```diff
@@ -196,9 +177,7 @@ func Convert_api_DeploymentCauseImageTrigger_To_v1_DeploymentCauseImageTrigger(i
 }
 
 func autoConvert_v1_DeploymentConfig_To_api_DeploymentConfig(in *DeploymentConfig, out *deploy_api.DeploymentConfig, s conversion.Scope) error {
-	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
-		defaulting.(func(*DeploymentConfig))(in)
-	}
+	SetDefaults_DeploymentConfig(in)
```

Comment:
You also need to update our custom conversions (conversions.go in the same directory). You actually need to go over all custom conversions in the repo and make sure we are defaulting correctly.